pkgs/desktops/budgie/budgie-control-center/default.nix

```diff
@@ -0,0 +1,127 @@
+{ lib
+, stdenv
+, substituteAll
+, fetchFromGitHub
+, accountsservice
+, budgie-desktop
+, clutter-gtk
+, clutter
+, colord-gtk
+, colord
+, cups
+, gcr
+, gettext
+, glib
+, glibc
+, gnome-desktop
+, gnome-online-accounts
+, gnome
+, gsound
+, gtk3
+, ibus
+, krb5
+, libepoxy
+, libgnomekbd
+, libgtop
+, libgudev
+, libhandy
+, libnma
+, libpulseaudio
+, libpwquality
+, libsecret
+, libwacom
+, libxml2
+, meson
+, modemmanager
+, networkmanager
+, networkmanagerapplet
+, ninja
+, pkg-config
+, polkit
+, samba
+, shadow
+, tzdata
+, udisks
+, upower
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "budgie-control-center";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "BuddiesOfBudgie";
+    repo = pname;
+    rev = "v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-z9apestNLEUKzrCMNo0BNAWeyE6FsUCAzcHIom8LcUs=";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./paths.patch;
+      budgie_desktop = budgie-desktop;
+      gcm = gnome.gnome-color-manager;
+      inherit cups glibc libgnomekbd shadow;
+      inherit networkmanagerapplet tzdata;
+    })
+  ];
+
+  nativeBuildInputs = [
+    gettext
+    glib
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    accountsservice
+    clutter
+    clutter-gtk
+    colord
+    colord-gtk
+    gcr
+    gnome-desktop
+    gnome-online-accounts
+    gnome.cheese
+    gnome.gnome-bluetooth_1_0
+    gnome.gnome-settings-daemon
+    gsound
+    gtk3
+    ibus
+    krb5
+    libepoxy
+    libgtop
+    libgudev
+    libhandy
+    libnma
+    libpulseaudio
+    libpwquality
+    libsecret
+    libwacom
+    libxml2
+    modemmanager
+    networkmanager
+    polkit
+    samba
+    udisks
+    upower
+  ];
+
+  preConfigure = ''
+    # For ITS rules
+    addToSearchPath "XDG_DATA_DIRS" "${polkit.out}/share"
+  '';
+
```

Comment:
Switching sound effects doesn't work because the audio files are somewhere else. Check `gnome-control-center`.

Reply:
Sound effects are in /usr/share/sounds/budgie/default/alerts

Reply:
Thanks 😁